config/sections/mixins/headline.php

```diff
@@ -6,9 +6,16 @@
     'props' => [
         /**
          * The headline for the section. This can be a simple string or a template with additional info from the parent page.
+         * @todo deprecate in 3.8
```

Comment:
Let's already deprecate it in 3.7 - we can still keep it around for longer and remove it in 3.9 or later. But to already give the signal (just as deprecated code comment and release notes) in 3.7